insights/parsers/systemd/config.py

```diff
@@ -17,58 +18,55 @@ class SystemdDocker(Parser):
 
     The content of file ``/usr/lib/systemd/system/docker.service`` is recorded
     via INI format, ``ConfigParser`` could be used to parse the content.
+    
+    Typical output of the ``systemd_docker`` command is::
+    
```

Comment:
@wushiqinlou if you remove the extra whitespace on some of the lines you can eliminate the flake8 errors.  You can check this before you submit the patch by typing the `flake8` command in the root dir of your insights-core project.